ui/console/src/main/scripts/plugins/metrics/plugins/metrics/html/response-time.html

```diff
@@ -57,7 +60,7 @@ <h3 class="pull-left">Metrics</h3>
       </div>
     </div>
 
-    <div class="panel panel-default hk-graph">
+    <div class="panel panel-default hk-graph" ng-if="vm.average >= 0">
```

Comment:
Yeah, I'll buy that. Just have to ask why ng-if and you have a legitimate reasoning behind use of it :+1: